create-job-ui/src/pages/CreateJobPage.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react'
+import { Calander } from '../components/Calander';
+import { DropDownMenu } from '../components/DropDownMenu'
+import { DropDownSearch } from '../components/DropDownSearch';
+import { ListOfCheckboxes } from '../components/ListOfCheckboxes';
+import { ListOfOptions } from '../components/ListOfOptions';
+import { SimpleDropDown } from '../components/SimpleDropDown';
+import { Textfeild } from '../components/Textfeild'
+import './CreateJobPage.css';
+export const CreateJobPage = () => {
+
+  const [jobTitle, setJobTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [department, setDepartment] = useState('');
+  const [responsibilities, setResponsibilities] = useState([]);
+  const [degrees, setDegrees] = useState([]);
+  const [employmentCategories, setEmploymentCategories] = useState([]);
+  const [genders, setGenders] = useState([]);
+  const [location, setLocation] = useState('');
+  const [softskills, setSoftskills] = useState([]);
+  const [technicalskills, setTechnicalskills] = useState([]);
+  const [experienceLevel, setExperienceLevel] = useState([]);
+  const [perksAndBenefits, setPerksAndBenefits] = useState([]);
+  const [travelling, setTravelling] = useState([]);
+  const [vacancies, setVacancies] = useState('');
+  const [closingDate, setClosingDate] = useState(null);
+
+
+  const [departmentOptions, setDepartmentOptions] = useState(['Cloud Native Development', 'Data Engineering']);
```

Comment:
@asharMehmood2211004KHICND note that we only need to use `useState` if the value is changed or updated, so it looks like it's not needed here